pkgs/development/compilers/go/1.17.nix

```diff
@@ -40,9 +40,14 @@ let
     "armv5tel" = "arm";
     "armv6l" = "arm";
     "armv7l" = "arm";
+    "mips" = "mips";
```

Comment:
Was this removed in go_1_17?

Reply:
No. It's still in master: https://github.com/golang/go/blob/8ff254e30ba9d9c13747fe42213c3f54b47c58e7/src/cmd/dist/build.go#L62